lib/_tls_wrap.js

```diff
@@ -967,7 +967,7 @@ exports.connect = function(/* [port, host], options, cb */) {
     minDHSize: 1024
   };
 
-  options = util._extend(defaults, options || {});
+  options = Object.assign(defaults, options || {});
```

Comment:
Good point. With one exception, this was mainly a find/replace. I'll go through it once more to clean up this kind of usage.